.github/workflows/check.yml

```diff
@@ -26,21 +26,19 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          submodules: 'recursive'
+          submodules: "recursive"
```

Comment:
you should check your editor and remove these auto-fix rules that you have, it's generally not good manners to go editing things unrelated to what you're intending to change without good reason to do so

Reply:
For some reason, my linter was automatically replacing the quotes, fixed it